composer.json

```diff
@@ -10,10 +10,10 @@
         }
     ],
     "require": {
-        "php": "^8.0",
-        "illuminate/support": "^9.0",
-        "illuminate/console": "^9.0",
-        "illuminate/database": "^9.0"
+        "php": "^8.3",
+        "illuminate/support": "^9.0|^11.0",
```

Comment:
I did a typo `|` should be `||` according to composer docs (though `|` still works)

> A double pipe (||) will be treated as a logical OR

> Note: In older versions of Composer the single pipe (|) was the recommended alternative to the logical OR. Thus for backwards compatibility the single pipe (|) will still be treated as a logical OR.